spce/ibi_lammps/spce.in

```diff
@@ -5,16 +5,18 @@ atom_style	atomic
 
 read_data       spce.data
 
-velocity	all create 2.49 87287
+velocity	all create 2.49435 87287
 
 pair_style	table linear 1000
 pair_coeff	1 1 CG_CG.pot VOTCA 0.9
 
-timestep	0.001
+timestep	0.002
```

Comment:
I changed it to make it consistent to the GROMACS ibi tutorial where the timestep is 2 fs.